### DISCUSSION TITLE
Customization of security onion

### DISCUSSION THREAD

**Discussion by peterchain7** [Security-Onion-Solutions/securityonion#8711] [General] [answered]
I was customizing my security onion overview page by `sudo cp /opt/so/saltstack/default/salt/soc/files/soc/motd.md /opt/so/saltstack/local/salt/soc/files/soc/` and editing `motd.md`.  But images can't be displayed in an overview page. I used Markdown image styles. What can i do?

**Comment by dougburks** [comment, marked as answer]
This seems to work for me:
```
![Security Onion Solutions](https://securityonionsolutions.com/_nuxt/img/sos-white.31689c0.svg)
```

![image](https://user-images.githubusercontent.com/1659467/189644696-a6ca6fd0-ffde-4f26-9c82-eee08c32a0fd.png)

**Reply by peterchain7** [reply]
in which directory did you put your image? for me online images works, But i don't know the directory to put  image i want to display in overview page

**Reply by dougburks** [reply]
I didn't place my image in a local directory, I simply referenced an image on another webserver (in this case https://securityonionsolutions.com):
```
![Security Onion Solutions](https://securityonionsolutions.com/_nuxt/img/sos-white.31689c0.svg)
```

**Comment by peterchain7** [comment]
What about in local directory?
